COPYRIGHT.txt

```diff
@@ -485,7 +485,7 @@ License: Apache-2.0
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
  .
-     http://www.apache.org/licenses/LICENSE-2.0
+     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
We should not change the "copying" files. That's the suggested url in the official page:
https://www.apache.org/licenses/LICENSE-2.0#apply